src/network/networkd-can.c

```diff
@@ -89,7 +89,7 @@ int can_set_netlink_message(Link *link, sd_netlink_message *m) {
                         return r;
         }
 
-        if (link->network->can_restart_us > 0) {
+        if (link->network->can_restart_us_set) {
```

Comment:
It is not necessary to introduce the flag. Please simply drop the `if` condition, and set the attribute unconditionally.

Reply:
I'd considered that at first, but wasn't sure what the intended semantics were for an unset value (actively push the value to the kernel vs do nothing). I'll go ahead and make that change.